components/SearchResultDetails.vue

```diff
@@ -59,6 +59,13 @@
                     </svg>
                     {{ phone }}
                 </div>
+                <div class="email flex my-4" v-if="email !== 'none' && email !== 'email@email.com'">
```

Comment:
Just curious, where are `none` and `email@email.com` coming from? Are these placeholders? If so, it's better to use variables instead of hardcoded strings in case these can change in the future.

Reply:
I've added an array of the placeholders. That way if we find more, we can just add them to the list.